File: 2737-row-with-maximum-ones/2737-row-with-maximum-ones.js
/**
 * @param {number[][]} mat
 * @return {number[]}
 */
var rowAndMaximumOnes = function(mat) {
    let res = [-1,-1] ;
    let i = 0 , j = 0;
    let n = mat.length;
    let m = mat[0].length;
    for( i=0 ; i < n ; i++){
        let c = 0;
        for (j = 0 ; j < m ; j++ ){
            if (mat[i][j] == 1){
                c++;
            }
        }
        if(c > res[1]) {
            res[1] = c ;
            res[0] = i;
        }
    }
    return res;
     
    
};

File: 2737-row-with-maximum-ones/2737-row-with-maximum-ones.py
class Solution(object):
    def rowAndMaximumOnes(self, mat):
        """
        :type mat: List[List[int]]
        :rtype: List[int]
        """
        res = [float("-inf"),float("-inf")]
        for i in range(len(mat)):
            c=0
            for j in range(len(mat[i])):
                if mat[i][j] == 1 :
                    c+=1
            if c > res[1]:
                res[1] = c
                res[0] = i
        return res


        